sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/ChangeFeedOffset.scala

```diff
@@ -34,11 +34,11 @@ private case class ChangeFeedOffset
   override def json(): String = jsonPersisted
 }
 
-private object ChangeFeedOffset {
+private[spark] object ChangeFeedOffset {
   private val IdPropertyName: String = "id"
   private val StatePropertyName: String = "state"
   private val InputPartitionsPropertyName: String = "partitions"
-  private val V1Identifier: String = "com.azure.cosmos.spark.changeFeed.offset.v1"
+  val V1Identifier: String = "spark.cosmos.changeFeed.offset.v1"
```

Comment:
is this stored anywhere? what happens if the customer upgrades the spark connector.

does this cause any disruption in continuating the process started on earlier version of the spark connector?

Reply:
I just read the description of the PR. makes sense. thanks.

Reply:
In theory it would - but because any attempt to continue the stream fails with the error trying to read/parse/deserialize the offset anyway in this case it doesn't hurt.

Reply:
Below we are comparing with `==`:

`parsedNode.get(IdPropertyName).asText("") == V1Identifier`

Shouldn't the comparisson change?